lib/widget/app_text.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:pokedex_start/utilities/constant.dart';
+import 'package:flutter/material.dart';
+
+class BodySmallText extends StatelessWidget {
+  const BodySmallText({
+    super.key,
+    required this.text,
```

Comment:
make it as a habit to put required properties on top of non-required properties 🙂

Reply:
any update on this part? 😅